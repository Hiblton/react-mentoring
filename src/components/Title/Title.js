const Title = ({ title, color = "white" }) => (
  <h1 style={{ color, fontWeight: 100 }} title={title}>
    {title}
  </h1>
);

export default Title;
