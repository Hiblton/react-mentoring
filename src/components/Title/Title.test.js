import React from "react";
import { render, cleanup } from "@testing-library/react";
import { Title } from "./Title";

afterEach(cleanup);

it("should take a snapshot", () => {
  const { asFragment } = render(<Title title="title" />);

  expect(asFragment()).toMatchSnapshot();
});
