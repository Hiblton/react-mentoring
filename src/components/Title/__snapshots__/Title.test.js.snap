// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <h1
    style="color: white; font-weight: 100;"
    title="title"
  >
    title
  </h1>
</DocumentFragment>
`;
